addons/hr_holidays/models/hr_leave_type.py

```diff
@@ -94,8 +94,15 @@ def _model_sorting_key(self, leave_type):
             #       by the Time Off Officer, Employee's Approver or both.""")
 
     has_valid_allocation = fields.Boolean(compute='_compute_valid', search='_search_valid', help='This indicates if it is still possible to use this type of leave')
-    time_type = fields.Selection([('other', 'Worked Time'), ('leave', 'Absence')], default='leave', string="Kind of Time Off",
-                                 help="The distinction between working time (ex. Attendance) and absence (ex. Training) will be used in the computation of Accrual's plan rate.")
+    time_type = fields.Selection([
+        ('work', 'Paid, Worked Time'),
+        ('paid', 'Paid, non-Worked Time'),
+        ('unpaid', 'Unpaid, non-Worked Time')
+        ], default='unpaid', string="Kind of Time Off",
```

Comment:
I would suggest that the time off are paid by default, seems like the most common use case.
```suggestion
        ], default='paid', string="Kind of Time Off",
```